server/bot/commands/api.ts

```diff
@@ -224,10 +224,22 @@ export const handleDog: Command = {
       // Fetch a random picture for a sub-breed.
     } else if (args[0] && args[1]) {
       try {
-        const { message } = await (await fetch(
+        let { message } = await (await fetch(
           `http://dog.ceo/api/breed/${args[0].toLowerCase()}/${args[1].toLowerCase()}/images/random`
```

Comment:
There are no dogs with spaces, it should always be joined.
```suggestion
        const { message } = await (await fetch(
          `http://dog.ceo/api/breed/${args.join('').toLowerCase()}/${args[1].toLowerCase()}/images/random`
```